cogs/track_react.py

```diff
@@ -88,6 +88,18 @@ def parse_codeblocks(message: str) -> str:
                 lines.append(line)
         return "\n".join(lines)
 
+    @staticmethod
+    def parse_url(message: str) -> str:
+        """Return a message with URLs removed"""
+        lines = []
+        pattern = r"(https?://\S+)"
+        message = re.sub(pattern, "", message)
```

Comment:
Is there a reason the sub needs to happen both on a whole-message basis and a second time on a per line basis?